vNext/extensions/applicationinsights-react-js/rollup.config.js

```diff
@@ -11,45 +12,6 @@ const banner = [
   " */"
 ].join("\n");
 
-const browserRollupConfigFactory = isProduction => {
```

Comment:
Why remove browser entirely?

Reply:
It was not being referenced in package.json, so to avoid confusion and bloating the package.

Reply:
added it back for consistency with other extensions. Also to enable scenarios where one might want to self host browserified version